pom.xml

```diff
@@ -14,7 +14,7 @@
     </properties>
 
 
-   <dependencies>
+    <dependencies>
         <dependency>
             <groupId>org.mockito</groupId>
```

Comment:
I will find this resource very useful for my personal projects. Thanks for showing me.